### PR TITLE
PT-59459679 SC WS Consistent broken encoding responses

### PR DESCRIPTION
**Pull request by cytadela8**
[Pivotal](https://www.pivotaltracker.com/story/show/159459679)
Protocol PR: aeternity/protocol#354

### CODE REVIEW

**Comment by cytadela8** [comment]
Tests might fail due to CircleCI problem - "Unauthorized" error on docker tests.

File: apps/aehttp/src/sc_ws_api_jsonrpc.erl [hunk 1]
```diff
@@ -190,14 +190,18 @@ error_data_msgs() ->
...
+     , 1006 => <<"Broken encoding: contract pubkey">>
      , 1007 => <<"Contract init failed">>
      , 1008 => <<"Not a number">>
+     , 1009 => <<"Broken encoding: contract bytearray">>
+     , 1010 => <<"Broken encoding: transaction">>
      }.
 
-broken_encoding_code(accounts ) -> 1005;
-broken_encoding_code(contracts) -> 1006.
+broken_encoding_code(account   ) -> 1005;
```
Comment (velzevur) [inline apps/aehttp/src/sc_ws_api_jsonrpc.erl]:
```suggestion
broken_encoding_code(account    ) -> 1005;
```

File: apps/aehttp/src/sc_ws_api_jsonrpc.erl [hunk 2]
```diff
@@ -190,14 +190,18 @@ error_data_msgs() ->
...
+broken_encoding_code(contract  ) -> 1006;
```
Comment (velzevur) [inline apps/aehttp/src/sc_ws_api_jsonrpc.erl]:
```suggestion
broken_encoding_code(contract   ) -> 1006;
```

File: apps/aehttp/test/aehttp_integration_SUITE.erl [hunk 3]
```diff
@@ -6003,6 +6043,8 @@ data_code_to_reason([1006])      -> <<"broken_encoding: contracts">>;
 data_code_to_reason([1007])      -> <<"contract_init_failed">>;
 data_code_to_reason([1008])      -> <<"not_a_number">>;
 data_code_to_reason([1005,1006]) -> <<"broken_encoding: accounts, contracts">>;
```
Comment (lucafavatella) [inline apps/aehttp/test/aehttp_integration_SUITE.erl]: When is this exercised?

Comment (cytadela8) [reply apps/aehttp/test/aehttp_integration_SUITE.erl]: On line 4750

Comment (lucafavatella) [reply apps/aehttp/test/aehttp_integration_SUITE.erl]: I see this is test code only - sorry for the noise. Thanks.

Comment (cytadela8) [reply apps/aehttp/test/aehttp_integration_SUITE.erl]: https://github.com/aeternity/aeternity/blob/172733506a526b688eea795090b1158694c9581e/apps/aehttp/test/aehttp_integration_SUITE.erl#L4750

**Comment by cytadela8** [comment]
Removed empty endline in release note file.

**Comment by cytadela8** [comment]
force push to retrigger AppVeyor

**Comment by cytadela8** [comment]
CircleCI passed fully before the force push (to retrigger AppVeyor). AppVeyor errors don't seem to make sense, but I don't understand what AppVeyor does exactly.

**Comment by cytadela8** [comment]
One more try at restarting Appveyor

**Comment by lucafavatella** [comment]
> One more try at restarting Appveyor

It seems to fail in master. We will merge without the Win CI passing, we shall investigate this but it is independent.

**Pull request merged**